src/main/java/spoon/reflect/declaration/CtFormalTypeDeclarer.java

```diff
@@ -27,27 +27,57 @@
 public interface CtFormalTypeDeclarer extends CtElement {
```

Comment:
This abstract element defines a declaration that accepts formal type
 * parameters (aka generics), such as a CtType (<code>class A<E></code>), CtMethod or CtConstructor